Sources/Utility/Git.swift

```diff
@@ -57,7 +57,6 @@ public class Git {
         return knownVersions
     }
 
-    public class var tool: String {
-        return "git"
-    }
+    /// A shell command to run for Git. Can be either a name or a path.
+    public static var tool: String = "git"
```

Comment:
I had a very brief talk with @abertelrud and I think I understand the intended usage. The PATH will probably be a temporary fix rather than a permanent override.

Since the overrides are expected to be paths, I wonder if its better to use `AbsolutePath` for them. Something like:

```swift

public class Git {

    public static var tool: String {
         return toolOverride.asString ?? "git"
    }
    
    public static var toolOverride: AbsolutePath? = nil
}

```

We could also add that part later if needed. Either way, feel free to merge it.